kroxylicious/src/main/java/io/kroxylicious/proxy/internal/filter/FetchResponseTransformationFilter.java

```diff
@@ -38,11 +38,11 @@
  */
 public class FetchResponseTransformationFilter implements FetchResponseFilter {
 
-    public static class FetchResponseTransformationFilterConfig extends FilterConfig {
+    public static class FetchResponseTransformationBaseConfig extends BaseConfig {
```

Comment:
Again not a fan of `Base` in the type name. 

I also kinda think calling it `FetchResponseTransformationFilterConfig` works as it is the config for that filter.